pom.xml

```diff
@@ -156,7 +161,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-branch-source</artifactId>
-            <version>2.2.3</version>
+            <version>2.5.4-rc860.9c636841daa2</version>
```

Comment:
TODO: once the PR https://github.com/jenkinsci/github-branch-source-plugin/pull/235 is merged and released